### PR TITLE
REL-815: Show seconds for periods in timing constraints.

### PR DESCRIPTION
**Pull request by fnussber**
From the JIRA task: Requirements: Periods of timing windows must include seconds, so the format must be hhh:mm:ss. The default period must be 48:00:00. 'Nuff said.

### CODE REVIEW

File: bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/TimingWindowDialog.java [hunk 1]
```diff
@@ -89,9 +89,9 @@ public void insertUpdate(DocumentEvent e) {
 		};
 
 		window.setValue(new Date());
-		duration.setValue(MS_PER_HOUR * 123 + MS_PER_MINUTE * 45);
```
Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/TimingWindowDialog.java]: This is deleted code but I'm wondering what was that formula?

Comment (fnussber) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/TimingWindowDialog.java]: It just displayed "123.45". I don't think there was any particular reason for this.

Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/TimingWindowDialog.java]: Excellent!

Comment (swalker2m) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/TimingWindowDialog.java]: Guessing it was a way to indicate that fractions of an hour are acceptable?

**Comment by swalker2m** [comment]
:+1:

File: bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/TimingWindowDialog.java (hunk 1 above)
Comment (fnussber) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/TimingWindowDialog.java]: Yes, that's what I think, too.

**Pull request merged**